Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/storage/entity/ContactDiaryLocationEntity.kt

```diff
@@ -19,10 +20,13 @@ data class ContactDiaryLocationEntity(
         get() = locationId
 }
 
+private const val MAX_CHARACTERS = 250
+private fun String.trimMaxCharacters(): String = this.trim().trimToLength(MAX_CHARACTERS)
```

Comment:
Afaik, we shall not trim the input values. Leading and trailing whitespaces for instance shall be possible (no input validation + apparently this also allowed on most phone contact apps)

Reply:
removed it